libraries/util/appCommon/adminKpc.py

```diff
@@ -1244,4 +1244,157 @@ def searchCustomer(page, strEmail):
+        
+        param strValue: Text
+        returns: None
+        Author: abernal_20240318
+        """
+        uCommon.waitAndClickElem(page, pAdmin.cu.totalRewardEditBtn)
+        uCommon.waitAndSetElem(page, pAdmin.cu.rewardEditLbl, strValue)
+        uCommon.waitAndClickElem(page, pAdmin.cu.editEnterBtn)
+        uCommon.waitElemToBeVisible(page, pAdmin.cu.confirmationModalLbl)
+        uCommon.waitAndClickElem(page, pAdmin.cu.confirmationYesBtn)
+        uCommon.wait(page, 2)
+        
+    @uCommon.ufuncLog  
+    def editTotalCreditsValue(page, strValue):
+        """ 
+        Objective: To edit the Total Credits value.
+        
+        param strValue: Text
+        returns: None
+        Author: abernal_20240318
+        """
+        uCommon.waitAndClickElem(page, pAdmin.cu.totalCreditEditBtn)
+        uCommon.waitAndSetElem(page, pAdmin.cu.creditEditLbl, strValue)
+        uCommon.waitAndClickElem(page, pAdmin.cu.editEnterBtn)
+        uCommon.waitElemToBeVisible(page, pAdmin.cu.confirmationModalLbl)
+        uCommon.waitAndClickElem(page, pAdmin.cu.confirmationYesBtn)
+        
+    @uCommon.ufuncLog  
+    def getTotalRewardsValue(page, strTotalRewards):
+        """ 
+        Objective: To get the Total Rewards value.
+        param: strTotalRewards: Text
+        returns strTotalRewards: Text
+        Author: abernal_20240319
+        """
+        intTotalRewards = uCommon.getElemText(page, pAdmin.cu.totalRewardValueLbl)
+        return intTotalRewards
+    
+    @uCommon.ufuncLog  
+    def getTotalCreditsValue(page, strTotalCredits):
+        """ 
+        Objective: To get the Total Credits value.
+        param strTotalCredit: Text
+        returns strTotalCredit: Text
+        Author: abernal_20240319
+        """
+        strTotalCredits = uCommon.getElemText(page, pAdmin.cu.totalCreditValueLbl)
+        return strTotalCredits
+    
+    @uCommon.ufuncLog  
+    def deductTotalRewardValue(page, strValue):
+        """ 
+        Objective: To deduct rewards from the Total Rewards.
+        param strValue: Text
+        returns: None
+        Author: abernal_20240319
+        """
+        strTotalReward = uCommon.getElemText(page, pAdmin.cu.totalRewardValueLbl)
+        uCommon.waitAndClickElem(page, pAdmin.cu.totalRewardEditBtn)
+        strValue = int(strValue) + 100
+        strNewValue = '-' + str(strValue)
+        uCommon.waitAndSetElem(page, pAdmin.cu.rewardEditLbl, strNewValue)
+        uCommon.waitAndClickElem(page, pAdmin.cu.editEnterBtn)
+        uCommon.waitElemToBeVisible(page, pAdmin.cu.confirmationModalLbl)
+        uCommon.waitAndClickElem(page, pAdmin.cu.confirmationYesBtn)
+        uCommon.waitElemToBeVisible(page, pAdmin.cu.confirmationDeductionModalLbl)
+        uCommon.waitAndClickElem(page, pAdmin.cu.closeConfirmationDeductBtn)
+        uCommon.waitAndClickElem(page, pAdmin.cu.cancelEditBtn)
+        strTotalReward = uCommon.getElemText(page, pAdmin.cu.totalRewardValueLbl)

```

Comment:
Line 1338 and line 1349, same lang ng ginagawa, better if gawan na lang ng function, ireturn na lang strTotalReward.